articles/ai-services/speech-service/text-to-speech-avatar/what-is-text-to-speech-avatar.md

```diff
@@ -27,33 +27,46 @@ With text to speech avatar's advanced neural network models, the feature empower
 Text to speech avatar capabilities include:
 
 - Converts text into a digital video of a photorealistic human speaking with natural-sounding voices powered by Azure AI text to speech.
-- Provides a collection of standard avatars.
+- Provides a collection of standard avatars, see [Standard avatars](./standard avatars.md)for a full list of supported standard avatars.
 - Azure AI text to speech generates the voice of the avatar. For more information, see [Avatar voice and language](#avatar-voice-and-language).
 - Synthesizes text to speech avatar video asynchronously with the [batch synthesis API](./batch-synthesis-avatar.md) or in [real-time](./real-time-synthesis-avatar.md).
 - Provides a [content creation tool](https://speech.microsoft.com/portal/talkingavatar) in Speech Studio for creating video content without coding.
 - Enables real-time avatar conversations through the [live chat avatar tool](https://speech.microsoft.com/portal/livechat) in Speech Studio.
 
-With text to speech avatar's advanced neural network models, the feature empowers you to deliver lifelike and high-quality synthetic talking avatar videos for various applications while adhering to responsible AI practices.
 
 ## Avatar voice and language
 
 You can choose from a range of standard voices for the avatar. The language support for text to speech avatar is the same as the language support for text to speech. For details, see [Language and voice support for the Speech service](../language-support.md?tabs=tts). Standard text to speech avatars can be accessed through the [Speech Studio portal](https://speech.microsoft.com/portal/talkingavatar) or via API.
 
 The voice in the synthetic video could be an Azure AI Speech standard voice or the [custom voice](../custom-neural-voice.md) of voice talent selected by you.
 
+## Avatar type
+- Video Avatar: The avatar is generated using a neural network model and requires a video recording for training. It supports half-body and full-body representations.
```

Comment:
The avatar is generated using a fine tuned model with a video recording for fine tuning.